.github/workflows/main.yml

```diff
@@ -13,7 +13,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "pypy-3.8", "pypy-3.9", "pypy-3.10"]
+        python-version:
+          [
+            "3.8",
+            "3.9",
+            "3.10",
+            "3.11",
+            "3.12",
+            "pypy-3.8",
+            "pypy-3.9",
+            "pypy-3.10",
+          ]
```

Comment:
This could be a lot simpler, even in multiline mode:

```yaml
- 3.8
- 3.9
- "3.10"
- 3.11
- 3.12
- pypy-3.8
- pypy-3.9
- pypy-3.10
```
Please use a non-JSON YAML list or revert this change.